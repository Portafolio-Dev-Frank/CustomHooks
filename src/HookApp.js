import React from 'react'

export const HookApp = () => {
    return (
        <div>
            <h1>hola a todos</h1>
        </div>
    )
}
