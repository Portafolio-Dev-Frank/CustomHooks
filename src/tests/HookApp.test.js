import { shallow } from 'enzyme'
import React from 'react'
import { HookApp } from '../HookApp'

describe('pruebas en HookApp', () => {
    test('debe de mostrar correctamente HookApp ', () => {
        const wrapper = shallow(<HookApp></HookApp>)
        expect(wrapper).toMatchSnapshot();
    })

})
