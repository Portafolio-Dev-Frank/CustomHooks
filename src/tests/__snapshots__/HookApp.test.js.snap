// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en HookApp debe de mostrar correctamente HookApp  1`] = `
<div>
  <h1>
    hola a todos
  </h1>
</div>
`;
